extensions/openfortivpn/src/connect.tsx

```diff
@@ -0,0 +1,43 @@
+import { Form, ActionPanel, Action, showToast, Toast } from "@raycast/api";
+import { promisify } from "util";
+import { exec as cExec } from "child_process";
+import { checkErrors } from "./utilities/common";
+
+const exec = promisify(cExec);
+
+interface CommandForm {
+  otp: string;
+}
+
+export default function Command() {
+  async function handleSubmit(values: CommandForm) {
+    const cmd = `/usr/bin/sudo /opt/homebrew/bin/openfortivpn${values.otp ? " -o " + values.otp : ""}`;
+    try {
+      await showToast({
+        style: Toast.Style.Success,
+        title: "Connecting",
+      });
+      await exec(cmd);
```

Comment:
This is really unsafe to use since it's allowing the user to modify the params, you should maybe look into spawn instead like [here](https://github.com/raycast/extensions/pull/1199/files#diff-63d79258fb1ff6bcf7dc2fcc723eeb747372b8eeb4c542b5a15d3fcae2a010e0R148)

Reply:
Hi @pernielsentikaer thanks a lot for your comments 💪
 I've made some improvements, hope it's better now

Reply:
Oh, i forgot write about it
Openfortivpn unfortunately in most cases requires sudo, so i've added ```useSudo``` preference to ```package.json``` file